sdk/gradlew.bat

```diff
@@ -0,0 +1,84 @@
+@if "%DEBUG%" == "" @echo off

```

Comment:
Can we rely on the Gradle files in the root folder?

Reply:
Should be able to, no problems with it so far.